### PR TITLE
Clean up the code for the next set of changes

### PR DESCRIPTION
**Pull request by teryhill**
Clean up the code for the next set of changes #1141

### CODE REVIEW

**Comment by teryhill** [comment]
@tmccormi and @aethelwulffe  ready for y'all to look at this

File: modules/calendar/add_edit_event.php [hunk 1]
```diff
@@ -57,25 +57,44 @@
...
+ $endtimem = '00';
+if (isset($_GET['endtimem'])) {
+  $endtimem = substr('00' . $_GET['endtimem'], -2);
+}
  //
  if (isset($_GET['starttimeh'])) {
   $starttimeh = $_GET['starttimeh'];
   if (isset($_GET['startampm'])) {
-   if ($_GET['startampm'] == '2' && $starttimeh < 12)
+        if ($_GET['startampm'] == '2' && $starttimeh < 12) {
```
Comment (aethelwulffe) [inline modules/calendar/add_edit_event.php]: This a 12 hour time format fix?  Do people still use 12 hour time format? 😜

File: modules/calendar/add_edit_event.php [hunk 2]
```diff
@@ -1912,7 +2048,7 @@ function SubmitForm() {
 
 $('#reasons_modal').iziModal({
                title: "<?php echo xlt('Reason for Cancellation'); ?>",
-               subtitle: "<?php echo xlt('Choose the reason for Cancellation. If the reason is not listed then please use the other'); ?>",
+               subtitle: "<?php echo xlt('Choose the reason for Cancellation.if the reason is not listed then please use other'); ?>",
```
Comment (aethelwulffe) [inline modules/calendar/add_edit_event.php]: 'Choose the reason for Cancellation.  If none are applicable, chose [Other]'

Comment (aethelwulffe) [reply modules/calendar/add_edit_event.php]: Additional typo introduced when fixing the language.

File: modules/calendar/add_edit_event.php (hunk 1 above)
Comment (teryhill) [reply modules/calendar/add_edit_event.php]: Yes

File: modules/calendar/add_edit_event.php (hunk 2 above)
Comment (teryhill) [reply modules/calendar/add_edit_event.php]: Change made

Comment (aethelwulffe) [reply modules/calendar/add_edit_event.php]: oops!  *choose* [Other]
I typodeded two.

Comment (aethelwulffe) [reply modules/calendar/add_edit_event.php]: Not seeing the change.  PUUUUSH

**Comment by teryhill** [comment]
I am finished updating this for now. I have some more planned but this needs reviewed and committed before I do any more.

**Comment by teryhill** [comment]
@tmccormi  Please review this. It needs to be committed before the next phase can be done.

**Comment by aethelwulffe** [comment]
@pri2si17-1997 @Perseus @alpinus4 @Trodrige and eveyone else....PLEEEZE do a review occasionally!  Frequent reviews ===eventual write access to the repo, and other folks getting to retire!

**Comment by teryhill** [comment]
Tired of waiting.

**Pull request merged**